lib/cors.js

```diff
@@ -12,6 +12,10 @@ const basics = opts => ({
   'access-control-allow-origin': opts.origin || defs.origin
 })
 
+const corsifyError = opts => err => {
+  throw Object.assign(err, { headers: basics(opts) })
```

Comment:
I did think of this but was not confident that we were cool with this not being immutable.

Reply:
> Oh, and using Ramda to merge results in an object that isn't an Error, which has it's own class of problems.

Yeah, i noticed that 😢